code/modules/mob/mob_verbs.dm

```diff
@@ -83,6 +83,36 @@
 	M.key = key
 
 
+/// This is only available to mobs once they join EORD.
+/mob/proc/eord_respawn()
+	set name = "EORD Respawn"
+	set category = "OOC"
+
+	if(stat != DEAD)
+		to_chat(src, "You can only use this when your dead.")
+		return
+
+	var/list/spawn_types = pick(subtypesof(/mob/living/carbon/xenomorph) + list(/mob/living/carbon/human))
```

Comment:
Check what `subtypesof(/mob/living/carbon/xenomorph)`, because I think the list won't be what you expect.

Reply:
Yeah I thought about only including fun castes but took the easy way out.
I'll update this to be a list of valid combat xenos instead.